src/js/components/Select/stories/select.stories.js

```diff
@@ -65,6 +65,7 @@ class SimpleSelect extends Component {
             value={value}
             options={options}
             onChange={({ option }) => this.setState({ value: option })}
+            dropProps={{ align: { top: 'top', left: 'right' } }}
```

Comment:
I think this is weird for the default select since this is what the user will see first most likely, can we add this in a separate story to show this?

Reply:
yeah let me remove this. I was adding this as an example

Reply:
and your overflow in this case is expected. storybook renders an iframe inside the main body, so the calendar cannot overflow outside that.